libfastpm/gravity.c

```diff
@@ -449,6 +477,40 @@ fastpm_solver_compute_force(FastPMSolver * fastpm,
         nacc = 3;
     }
 
+    FastPMCosmology * cosmo = fastpm->config->cosmology;
+    /* Transfer delta_k to delta_m with adding delta_nu*/
+    /*Computes delta_nu from a CDM power spectrum.*/
+    if(cosmo->ncdm_linearresponse) {
+        FastPMPowerSpectrum dmps[1] ={0};
+        /* calculate the neutrino power spectrum */
+        fastpm_powerspectrum_init_from_delta(dmps, fastpm->lptpm, delta_k, delta_k);
```

Comment:
We probably also do this for printing the power spectra.

 I remember delta_k is created for fastpm->pm, not fastpm->lptpm. mixing them up may crash for certain configurations of force factors.

Reply:
Thanks